src/execution/operator/persistent/buffered_csv_reader.cpp

```diff
@@ -1223,6 +1215,10 @@ bool BufferedCSVReader::ReadBuffer(idx_t &start) {
 	}
 	idx_t read_count = file_handle->Read(buffer.get() + remaining, buffer_read_size);
 
+	if (first_read && read_count >= 3 && buffer[0] == '\xEF' && buffer[1] == '\xBB' && buffer[2] == '\xBF') {
+		read_count -= 3;
+		memmove(buffer.get(), buffer.get() + 3, read_count);
```

Comment:
Can't we increment the position here, instead of using `memmove` (which could potentially result in moving a semi-large amount of memory)?

Reply:
I wasn't sure how the position arithmetic was applied, if incrementing position is enough, so be it.